app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -126,6 +126,14 @@ def pulp3_api
         client
       end
 
+      def backend_service_type(repository)
+        if pulp3_support?(repository)
+          Actions::Pulp3::Abstract::BACKEND_SERVICE_TYPE
```

Comment:
Need to think on it more but I wonder if this constant should come from somewhere other than an action class. Probably a nitpick either way

Reply:
Yeah, i couldn't think of a better place, right now actions are the only ones utilizing it

Reply:
would it be better if this were a method instead of a constant?